README.md

```diff
@@ -6,18 +6,26 @@
       * [Usage](#usage)
          * [Basic configuration parameters](#basic-configuration-parameters)
          * [Example configurations](#example-configurations)
-            * [Kernel driver config](#kernel-driver-config)
-            * [Advanced kernel driver config](#advanced-kernel-driver-config)
-            * [DPDK userspace driver config](#dpdk-userspace-driver-config)
+            * [Basic config](#basic-config)
+            * [Extended config](#extended-config)
          * [Advanced configuration](#advanced-configuration)
       * [Contributing](#contributing)
 
 # Accelerated Bridge CNI plugin
-This plugin enables the configuration and usage of Accelerated Bridge VF networks in containers and orchestrators like Kubernetes.
+This plugin allows using Linux Bridge with hardware offloading in containers and orchestrators like Kubernetes.
+Accelerated Bridge CNI plugin requires NIC with support of [SR-IOV](http://blog.scottlowe.org/2009/12/02/what-is-sr-iov/)
+technology with VFs in switchdev mode and support of Linux Bridge offloading.
 
-Network Interface Cards (NICs) with [SR-IOV](http://blog.scottlowe.org/2009/12/02/what-is-sr-iov/) capabilities are managed through physical functions (PFs) and virtual functions (VFs). A PF is used by the host and usually represents a single NIC port. VF configurations are applied through the PF. With Accelerated Bridge CNI each VF can be treated as a separate network interface, assigned to a container, and configured with it's own MAC, VLAN IP and more.
+In switchdev mode, two net-devs exist for a single VF: VF itself and VF representor.
```

Comment:
two net-devs exist for VF netdevice and VF representor

Reply:
done